lldb/test/Shell/SymbolFile/DWARF/x86/dwp-separate-debug-file.cpp

```diff
@@ -139,6 +139,20 @@
 // RUN:   -o "target variable a" \
 // RUN:   -b %t | FileCheck %s
 
+// Now move the .debug and .dwp file into another directory so that we can use
```

Comment:
Are we able to create a stale dwp testcase to ensure lldb won't load mismatch dwo files in dwp after ignoring UUID?
For example, 
1. Build matching dwp for one target
2. Move target main executable to a different location
3. Modify source file and rebuild a new dwp which should not match the original saved executable
4. Try to debug the original saved binary which will load dwp but examining local variables should emit error due to mismatch dwo.

Reply:
That is already covered by other split DWARF testing, so it doesn't need to be covered here.